### DISCUSSION TITLE
Wiki for cleaner documentation

### DISCUSSION THREAD

**Discussion by spusuf** [Jip-Hop/jailmaker#39] [Ideas]
Can you enable the wiki function in github so I can start writing some documentation. The sections I can write are:

- Networking and the differences between types.
- Performance and Resource comparisons between kubernetes and systemd-nspawn
- How to set up storage
- Best practices (E.g how to set up users instead of root)
- Commonly used apps and services which benefit from specific configurations for systemd
- Compatibility (both TrueNAS and container images)
- Usage guide including both jailmaker and machinectl commands

**Comment by Jip-Hop** [comment]
Sounds good! Wiki was already enabled. I just made the first page: https://github.com/Jip-Hop/jailmaker/wiki

**Reply by spusuf** [reply]
Hey, do you know how to create a PR for the wiki? If not I give you permission to just copy paste the two pages I've done on my branch (https://github.com/spusuf/jailmaker/wiki) at the moment, and I'll let you know when I've done more (probably tomorrow).

**Reply by Jip-Hop** [reply]
Thanks for your effort! Looks good. 😄 But I'm not looking forward to maintain a wiki by copy pasting edits. Perhaps this process works: https://stackoverflow.com/a/11481887

If not then I rather have markdown files directly inside the repo (perhaps a docs subfolder).

**Comment by spusuf** [comment]
> If not then I rather have markdown files directly inside the repo (perhaps a docs subfolder).

This seems like the way to go, let me know when it's been done.

Either that or is there a way to give me permissions to edit the wiki only?

**Comment by Jip-Hop** [comment]
> This seems like the way to go, let me know when it's been done.

You can do this yourself. Clone the jailmaker repo, add an empty docs folder and add markdown files with the documentation you've written. You can then create a PR.

**Reply by spusuf** [reply]
Hey so thats been merged, but is there a way to sync that to the github wiki? Or will the readme just point to the docs?

**Reply by Jip-Hop** [reply]
Can you try to edit the wiki again? I think you now have the appropriate permissions.

**Reply by spusuf** [reply]
No, can't edit the wiki

**Reply by Jip-Hop** [reply]
Ah that's too bad. Well I think for now it would be easiest if the Readme would point to the docs folder but I don't have time to do this at the moment.
